posthog/api/property_definition.py

```diff
@@ -133,8 +111,6 @@ def get_queryset(self):
             "event_names": tuple(event_names or []),
             "names": names,
             "team_id": self.team_id,
-            "order_ids_first": order_ids_first,
-            "excluded_ids": excluded_property_ids,
```

Comment:
Have you thought about/is it relevant to think about backwards compatibility here even if they're deprecated? Seems like these API changes could cause regressions if people don't expect the api to change. (Also not sure if anyone is actually using this api but worth thinking about to make sure we don't create trouble)

Reply:
I don't expect people to be actually using these parameters since I made them specifically to solve for an edge case in the app where i needed to have some custom ordering/exclusions. However agree that its a good idea to sanity check this. I'm not sure where the best place to check usage of an api but a quick search returns nothing. https://sentry.io/organizations/posthog2/issues/?query=excluded_ids+order_ids_first&statsPeriod=14d